### DISCUSSION TITLE
template string not working in classes

### DISCUSSION THREAD

**Discussion by SamarthKadam** [tailwindlabs/tailwindcss#10830] [Help]
I was trying to design the page using Tailwind CSS. I was trying to apply this style below
className={`grid gap-2 grid-cols-[1fr,4fr] w-[100%] px-12 py-14  bg-gradient-to-b from-[#c52f57]  to-[${value}]`}
here value was set to empty. how can this be resolved?
Please Help me Out.

**Comment by wongjn** [comment]
Class names need to appear in full in source code, see https://tailwindcss.com/docs/content-configuration#dynamic-class-names.

**Reply by SamarthKadam** [reply]
Yes but my use case is something like the above-mentioned. Docs are about conditional rendering. 

As in my above case, I am trying to add color using value as a name with a hex code in it. How do I make available the values inside the classes

**Reply by wongjn** [reply]
You could try applying the dynamic color via the `style` attribute, like
```html
<div className="… to-[var(--to)]" style={{ '--to': value }}>
```

**Reply by SamarthKadam** [reply]
Thank you @wongjn
